resources/src/img/speech-bubble.svg

```diff
@@ -0,0 +1,9 @@
YMuD8OIk21WuHdS11gDiJfgtcAuys974lScqJZ4Hz4iSq6y3N67JQbl+LVs5Z2j5m1krKs3ZJkvRHLwEz4ySq+zq0VEodXlkRvwOYmdYYkiQ1mS2UD7knaew8tVIHWLRyzm/bx8w6DHhVmuNIktQEdgIXx0k0J60B6n5OvQufAL7TgHEkScqq7cB74yT6eZqDtKS58z0VC6UvAP+vUeNJkpQRG4G3VXMr1Uqlevh9T4tWzvlN+5hZq4E30cA3E5IkBbSG8ir3eY0YrOHlWiyU3gH8COjX6LElSWqgJygX+tJGDRhkxlwslGYCNwJDQ4wvSVLKFgFvjJPomUYO2oiFcvuJk+h24Fzg6RDjS5KUojuAGY0udAhU6gBxEt0PnAH8KlQGSZLqaBfwj5Rvobo2RIDgC9aKhVIL8Bngi0CfwHEkSarGKuDd9bh9ai2Cl3qnYqH0GuAnwNjQWSRJqsAdlAv92dBBgh1+31ecRHcBZwI3hc4iSVIP7AK+ALw+C4UOGZqpd+o4HH8Z8E+4Ol6SlE3LgQ/FSXRH6CB7ylypdyoWSqOBrwHvDJ1FkqQOW4AvA1+Nk2hr6DD7ymypdyoWSq8HvglMCp1FktSr/Qq4LE6ix0IHOZDMlzpAsVDqR3mF/BXAwMBxJEm9y5PAJ+IkujF0kINpilLvVCyUxgNXAm+nybJLkprOBuDfgC/GSbQpdJieaMpiLBZKJwF/AxRp4E1pJEm9wlrgKuAbcRK9GDpMJZqy1DsVC6XjgMuB9+MNYiRJtVkFfB34ZpxE60OHqUZTl3qnYqF0NBABH8Jz7pKkyjwD/DPw3TiJNocOU4tclHqnYqF0GOVD8u8FCoHjSJKyazvl1ew/BH4ZJ9G2wHnqIlelvqdioXQi8B7gUmBc4DiSpGxIgGuAn8RJtCZ0mHrLbal36rhC3bmUZ+8XAcPCJpIkNdjjwLXANXESLQ2cJVW5L/U9FQulNmAKMLPjcQ6eg5ekvFlN+SYrtwO3x0n0SOA8DdOrSn1fHRe1OZs/lvxUXEUvSc1mHTCPjhIHFsVJtDtspDB6danvq2MmfxxwQhePIwJGk6TebhfwBLAEWLrnI06ilSGDZYml3kPFQmkYMBEYQfnucUOBIXv8uvPRH/9epTxpA04DTgwdJEfWALd1/Ho3sAlY381jDbA8TqItjY/aXCwfSeqBYqE0A/gBMD5wlDyYEyfReaFD5FFr6ACS1AziJJpHecb+vdBZcuDl0AHyylKXpB6Kk2h9nEQfBC4Ang+dp4lZ6imx1CWpQnES3QycCvw0dJYm9VLoAHllqUtSFTquRnYx8G7KH6lSzzlTT4mlLklVipNod5xE11Ketc8JnaeJWOopsdQlqUZxEj0DvBH4COWPZ6l7lnpKLHVJqoOOWfvVwOnA3aHzZJylnhJLXZLqKE6i5cB04HOUb++p/blQLiWWuiTVWZxEO+Mk+gpQAB4MnSeDnKmnxFKXpJTESbSQ8o2ivkL52uUqs9RTYqlLUoriJNoaJ9HnKB+SfzR0noyw1FNiqUtSA8RJdBflRXTfCp0lAyz1lHhDF0lqsGKhdB7wfWB06CyBtMVJ5OmIFDhTl6QGi5NoNuUL1vw4dJYA1lvo6bHUJSmAOInWxkl0KeVLzb4QOk8Deeg9RZa6JAUUJ9H1lGftN4XO0iCWeoosdUkKLE6i54ALgQ8AGwLHSZulniJLXZIyoOMys/8JnAbMC50nRV5NLkWWuiRlSJxEK4DXAZ8GtgaOkwZn6imy1CUpY+Ik2hUn0deAs4Dfh85TZ5Z6iix1ScqoOIkWA68G/gHYGThOvVjqKbLUJSnD4iTaHifR3wOvAZaGzlMHlnqKLHVJagJxEt0LTAa+ETpLjSz1FFnqktQk4iTaFCfRJ4E/AZ4KnadKrn5PkaUuSU0mTqLbgXbgB6GzVMGZeoosdUlqQnESvRQn0fuBtwKrA8ephKWeIktdkppYnEQ3Ur7M7P+EztJDlnqKLHVJanJxEq0C3ga8j+yXZtbzNTVLXZJyoOMys9dQPtf+69B5gRMWjQAAArRJREFUuuFCuRRZ6pKUI3ESPQnMAj4ObA4cpyvO1FPUEjqAJCkdxULpBOAaYGroLHvoEydRXq6OlznO1CUpp+IkWgqcA/wdsCNwHIANFnq6nKlLUi9QLJQmU561nxIwxso4icYEHD/3nKlLUi8QJ9HvgSlACdgdKIaL5FJmqUtSLxEn0ZY4iT4DvBZYESCCi+RSZqlLUi8TJ9E84HTgPxo8tKWeMktdknqhOInWx0n0IeAC4LkGDWupp8xSl6ReLE6imylfZvb6BgxnqafMUpekXi5OoheAdwLvAtalOJQL5VJmqUuSOi8zG1Oetc9OaRhn6imz1CVJr4iT6BngTcBfA5vqvHtLPWWWuiRpLx2z9m9TXiF/Vx13bamnzFKXJHUpTqLlwAzgcmBbHXZpqafMUpckHVCcRDvjJLoSKAAP1rg7F8qlzFKXJB1UnEQPUi72LwG7qtyNM/WUWeqSpB6Jk2hbnERXANOA5VXswlJPmaUuSapInER3A2cA36pwU0s9Zd56VZJUtWKhNAv4PtCTW6qOiJMozYvb9HrO1CVJVYuTaA7QDvy4B09fn3KcXs+ZuiSpLoqF0tuBbwOHdfHtDXESDW1wpF7HmbokqS7iJPop5cvM3tTFtz2f3gCWuiSpbuIkeg64EPhLYMMe37LUG8BSlyTVVcdlZr8PnAbM6/iypd4AlrokKRVxEq0AXgd8Gng+cBxJklQPxULJRXKSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmSJEmS1Cv9fy4N9Ru3kFN7AAAAAElFTkSuQmCC"/>
```

Comment:
uh oh - there shouldn't be bitmap data here, this seems to be because these images are actually bitmaps within Figma: https://www.figma.com/file/1RoHig9PBhi4nkDKdxMhHZ?node-id=113:0#84751128

I think Imogen will need to fix this